Installer-Linux/linux-hosts-installer.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
```

Comment:
The `[[` below is not part of POSIX (https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_04) and should be changed too IHMO.

Reply:
I have to admit it's certainly true but it does work in all shells that i tried. It can be fixed by changing the if-part to a `&&`.
But i would like to do that  in another PR, because i wanted to change something else on that line anyway. (See the pr that you'll receive in a couple of minutes)

Reply:
#607 fixes your square bracket objection (and improves the root check). But i would suggest not fixing it at all by ignoring #607 and only merging this one and #606 . See my comments in #607

Reply:
I'll change this in in #606

Reply:
Correction: I'll change this is in **yet** another PR. I don't know how this project works, but in a lot of projects the primar dev's don't like if you improve/fix more then 1 thing at a time in a PR...